shell/browser/electron_browser_main_parts_mac.mm

```diff
@@ -74,4 +76,23 @@
   [mainNib release];
 }
 
+std::string ElectronBrowserMainParts::GetCurrentSystemLocale() {
+  NSString* systemLocaleIdentifier =
+      [[NSLocale currentLocale] localeIdentifier];
+
+  // Mac OS X uses "_" instead of "-", so swap to get a real locale value.
+  std::string locale_value = [[systemLocaleIdentifier
+      stringByReplacingOccurrencesOfString:@"_"
+                                withString:@"-"] UTF8String];
+
+  // On disk the "en-US" resources are just "en" (http://crbug.com/25578), so
+  // the reverse mapping is done here to continue to feed Chrome the same values
+  // in all cases on all platforms.  (l10n_util maps en to en-US if it gets
+  // passed this on the command line)
+  if (locale_value == "en")
+    locale_value = "en-US";
```

Comment:
```suggestion

```

This is only needed when obtaining the application locale because chrome ships `en.lproj` instead of `en_US.lpoj` https://bugs.chromium.org/p/chromium/issues/detail?id=25578